src/Panels/Panels.js

```diff
@@ -1072,6 +1074,11 @@ module.exports = kind(
 				Spotlight.spot(this.$.breadcrumbs);
 				return true;
 			}
+		} else {
+			if (this.hasCloseButton && this.rtl === true && h && orig == h.$.inputDecorator) {
```

Comment:
It makes me uncomfortable that we are making an assumption about the component structure of the `Header` kind here (`h.$.inputDecorator`).